src/gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1597,6 +1608,7 @@ properties:
       - Induction Death
       - Induction Failure
       - None
+      - Presented With Mets
```

Comment:
A bio question, what is presented with mets? Google was useless unless I want to learn about baseball.

Reply:
Mets = Metastases, I never thought of how hard that would be to Google!